### DISCUSSION TITLE
One Form send to multiple Entities

### DISCUSSION THREAD

**Discussion by cube4d** [symfony/symfony#46787] [Q&A]
Hi,

i have two entities just as an example 

entity USER 
and 
entity COMPANY

now i want to have one form wich can handle both requests, so i have a field: name wich will be send to the entity user 
and then a field: company_name wich will be send to entity COMPANY

Is that possible ?  i read about the collection form thing but i dont know if it is the right way ?

Thanks

**Comment by ahundiak** [comment]
CollectionType is for when you have multiple instances of the same entity.  A one to many relation.  It's not suitable for this case.

But you can process unrelated entities in a single form by simply using an array.  Lets assume you have created a UserType as well as a CompanyType.  You can then do something like:
```
# your controller action
$user = new User();
$company = new Company();
$items = ['user' => $user, 'company' => $company];

$form = $this->createFormBuilder($items)
    ->add('user', UserType::class)
    ->add('company', CompanyType::class)
    ->add('save', SubmitType::class, ['label' => 'Do Something'])
     ->getForm();
```
And off you go.
